i915.c

```diff
@@ -525,7 +590,20 @@ static int i915_init(struct driver *drv)
 	i915->has_mmap_offset = gem_param(drv->fd, I915_PARAM_MMAP_GTT_VERSION) >= 4;
 	i915->has_fence_reg = gem_param(drv->fd, I915_PARAM_NUM_FENCES_AVAIL) > 0;
 
-	i915_bo_query_meminfo(drv, i915);
+	if (!i915_bo_query_prelim_meminfo(drv, i915)) {
+		drv_log("drv: Kernel does not support prelim\n");
+		i915_bo_query_meminfo(drv, i915);
+	} else {
+		drv_log("drv: kernel supports prelim\n");
```

Comment:
this log is not necessary. If kernel support prilim can be indicated in init function. Don't need print here.